src/applications/vaos/actions/appointments.js

```diff
@@ -375,7 +375,8 @@ export function confirmCancelAppointment() {
       });
       resetDataLayer();
     } catch (e) {
-      if (e?.errors?.[0]?.code === 'VAOS_400') {
+      const isVaos400Error = e?.errors?.[0]?.code === 'VAOS_400';
```

Comment:
It might safer to write a function to pull the error code out of an object, so that we accommodate both the FHIR format and the JSON API format. Should matter now because these are POST or PUT calls, but might save us time in the long run.